src/components/Header/Notifications/getNotifications.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBell } from '@fortawesome/free-solid-svg-icons';
+import { getNotificationsAction } from '../../../redux/actions/user/notifications';
+import { Button } from '../../common';
+import Modal from './Modal/Modal';
+import { getDataThunkPrivate } from '../../../redux/thunks';
+
+class GetNotifications extends Component {
+  state = {
+    showModal: false,
+    notification_number: 0,
+  };
+
+  closeModal = (e) => {
+    e.preventDefault();
+    this.setState({ showModal: false });
+  };
+
+  componentDidMount = async () => {
```

Comment:
Great work on this PR @shaluchandwani, Avoid making a lifecycle method asynchronous, I would rather you create an asynchronous function to fetch the notifications and invoke it in the lifecycle method.